filter:
  position_noise: 1.0
  attitude_noise: 0.1
  velocity_noise: 0.5
  bias_noise: 0.001

imu:
  publish_hz: 50
  sensor_pub_rate: 125
  topic: /imu/data
  noise:
    accel_noise_sigma: 0.00005
    gyro_noise_sigma: 0.0000024241
    accel_bias_rw_sigma: 0.07
    gyro_bias_rw_sigma: 0.15

odo:
  topic: /husky_velocity_controller/odom
  # topic: /odometry/filtered

slam:
  # topic: /blam/blam_slam/localization_integrated_estimate
  topic: /aft_mapped_to_init_high_frec

gicp_tran:
  topic: /gicp_localization/gicp_localization/initial_offset

frames:
  frame_id_out: map
  frame_id_fixed: greenhouse
  frame_id_imu: imu_link
  frame_id_odo: odom_link
